File: python/algorithms/palindrome.py
def reverse(s): 
    return s[::-1] 
  
def isPalindrome(s): 
    # Calling reverse function 
    rev = reverse(s) 
  
    # Checking if both string are equal or not 
    if (s == rev): 
        return True
    return False
  
  
# Driver code 
s = input()
ans = isPalindrome(s) 
  
if ans == 1: 
    print("Yes") 
else: 
    print("No") 

File: go/algorithms/palindrome.go
package main

import "fmt"

func isPalindrome(input string) bool {
	var x = len(input)
	for i := 0; i < x/2; i++ {
		if input[i] != input[len(input)-i-1] {
			return false
		}
	}
	return true
}

func main() {
	if isPalindrome("ARARA") {
		fmt.Println("Is a palindrome")
	}
}

